OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <cb n="b"/>
+                                    <hi rend="rubric">እሰግድ፡ ለኪ፡ ወእዌ</hi>ድሰኪ፡ ኦእግዝእትየ፡
+                                    <hi rend="rubric">ማርያም፡ በከመ፡ ወ</hi>ደሰኪ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡
+                                    በድርሳኑ፡ እንዘ፡ ይብል<hi rend="rubric">፨</hi> ተፈሥሒ፡ ኦእግዝእትየ፡ 
+                                    <hi rend="rubric">ማርያም፡</hi> ዘተሠገወ፡ ቃል፡ ወወፅአ፡ እምከርሥኪ፡ ከመ፡ መርዓዊ፡ 
+                                    ዘይወፅእ፡ እምጽርሑ<hi rend="rubric">፨</hi> ተፈሥሒ፡ ኦእግዝእትየ፡
+                                    <hi rend="rubric">ማርያም፡</hi> ዘወፅአ፡ እምኔኪ፡ ኖላዊ፡ ኄር፡ ዘበአማን፡ ዘኃሠሦ፡ ለበግዕ፡ ግዱፍ<hi rend="rubric">፨</hi>
+                                </incipit>
+                                
+                                <explicit xml:lang="gez">
+                                    <locus target="#28va"/>
+                                    ወተሰመይኪ፡ እመ፡ ወአመተ፡ በልደተ፡ መድኃኒነ፡ ቅዱስ<hi rend="rubric">፨</hi> ሰላም፡ ለኪ፡ ኦእግዝእትየ፡
+                                    <hi rend="rubric">ማርያም፡</hi> እስመ፡ ረከብኪ፡ ሞገሰ፡ በኀበ፡ እግዚአብሔር፡ <seg type="supplication">ጸልዪ፡ ወሰአሊ፡ ኀበ፡ ፍቁር፡
+                                    ወልድኪ፡ ኢየሱስ፡ ክርስቶስ፡ ከመ፡ ይስረይ፡ ለነ፡ ኃጣውኢነ<hi rend="rubric">፨</hi> <hi rend="rubric">፨</hi></seg>
+                                </explicit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i3.1.4">
+                                <locus from="28va" to="29rb"/>
+                                <title type="complete" ref="LIT6024IntroMM"/>
+                                <notatedMusic><desc>The hymn forming the second part of this text contains musical notation throughout.</desc></notatedMusic>
+                                <textLang mainLang="gez"/>
+                                
+                                <incipit xml:lang="gez">
+                                    <locus from="28va" to="28vb"/>
+                                    <hi rend="rubric">በፈቃደ፡ አቡሁ፡ ወበሥምረተ፡ ርእሱ፡ አ</hi>መ፡ ርእሶ፡ ይሰቅል<hi rend="rubric">፨</hi>
+                                    ወ<cb n="b"/>አመ፡ ተዓገሠ፡ ኵሎ፡ ሐፃውንተ፡ ቅዱስ፡ መስቀል<hi rend="rubric">፨</hi> ሶበ፡ ሰማዕኪ፡
+                                    <hi rend="rubric">ማርያም፡</hi> ዘነገሩኪ፡ በቃል። ዘውኅዘ፡ እምአዕይንትኪ፡ ከመ፡ ውኂዘ፡ ማይ፡ ዘይፈለፍል<hi rend="rubric">፨</hi>
+                                    ድንግል፡ ንዒ፡ በምሕረት፡ ወሣህል<hi rend="rubric">፨</hi>
+                                </incipit>
+                                
+                                <explicit xml:lang="gez">
+                                    <locus target="#29rb"/>
+                                    <hi rend="rubric">ወላዲተ፡ አምላክ፡ ማርያም፡ እንበለ፡ ሰብሳ</hi>ብ፡ ወሩካቤ፡ በል<hi rend="rubric">፨</hi>
+                                    ወእምዝ፡ ሰላም፡ ለኪ፡ <hi rend="rubric">ማርያም፡</hi> ኆኅተ፡ ምሥራቅ፡ ዘወልደ፡ ኖሬ፡ ሰላም፡ ለኪ፡ ዓፀድ፡ ዘኤስኬድሬ<hi rend="rubric">፨</hi>
+                                    ሰላም፡ ለኪ፡ ሥርጉት፡ በወራውሬ<hi rend="rubric">፨</hi> ሰ፡ ለ፡ እንተ፡ ወለድኪ፡ ፈጣሬ<hi rend="rubric">፨</hi>
```

Comment:
ሰ፡ ለ፡ is an abbreviated form of ሰላም፡ ለኪ፡. Do we want to encode this information?